bears/python/YapfBear.py

```diff
@@ -142,7 +142,8 @@ def run(self, filename, file,
         with prepare_file(options.splitlines(keepends=True),
                           None) as (file_, fname):
             corrected = FormatFile(filename,
-                                   style_config=fname)[0].splitlines(True)
+                                   style_config=fname,
+                                   verify=False)[0].splitlines(True)
```

Comment:
If this was a bug, and a major one - shouldn't there be a test case to avoid regression ?